### PR TITLE
Improve a little bit the code and improve the command history

### PR DESCRIPTION
**Pull request by Oseryx**
- Improve a little bit the code (``const`` and es6 arrow functions etc)
- Improve the command history system, instead of pushing we insert in the list a the index 1 (the index 0 is reserved to the current typed command) and then we increment and decrement the index (the default index is always 0 as it should be)

Have a nice day :))

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -11,13 +11,14 @@ function createWindow() {
...
         transparent: true,
-        icon: "../Icons/Terminal.ico",
+        icon: path.join(__dirname, 'Icons/Terminal.ico'),
         webPreferences: {
             nodeIntegration: true,
             enableRemoteModule: true,
             experimentalFeatures: true
         }
     });
+    mainWindow.webContents.openDevTools();
```
Comment (mellobacon) [inline src/main.js]: I think you left this here when testing lol

**Review by mellobacon — COMMENTED** [review]
Thanks for the PR. One question though. What are the changes in the package-lock.json about?

**Comment by Oseryx** [comment]
> Thanks for the PR. One question though. What are the changes in the package-lock.json about?

The main entry has been changed from dist/main.js to src/main.js as things should be

File: src/main.js (hunk 1 above)
Comment (Oseryx) [reply src/main.js]: fixed

**Review by mellobacon — APPROVED** [review]

**Pull request merged**